docs/content/guides/observability/ui/explore.md

```diff
@@ -0,0 +1,143 @@
+* **Visualize and monitor metrics**: With the built-in Prometheus integration, the Gloo UI has access to workload-specific metrics, such as the number of requests that were received for a workload. This data is visualized in the Gloo UI graph. 
+
+## Before you begin
+
+Follow the steps to [Set up the Gloo UI]({{< versioned_link_path fromRoot="/guides/observability/ui/setup/" >}}). 
+
+## Launch the UI
+
+1. Port-forward the Gloo UI pod. 
+   ```sh
+   kubectl port-forward deployment/gloo-mesh-ui -n gloo-system 8090
+   ```
+2. Open the Gloo UI dashboard. 
+   ```sh
+   open http://localhost:8090/dashboard
+   ```
+      
+   ![Gloo UI dashboard]({{< versioned_link_path fromRoot="/img/ui-dashboard.png" >}})
+   
+
+## Home
+
+View the health and performance of your Gloo Gateway control and data plane components, and view recommendations to harden your setup by using the **Dashboard** and **Insights** pages.
+
+### Dashboard
+
+The Gloo UI dashboard provides an at-a-glance overview of your Gloo Gateway setup and the health of your components. If you have gateways that use the Kubernetes Gateway API, you can also view request rates, failures, and request latency on the dashboard. Note the the request rate, failures, and latency is not available when you use gateways that are based on the Gloo Edge API. 
+
+![Gloo UI dashboard]({{< versioned_link_path fromRoot="/img/ui-dashboard.png" >}})
+
+### Insights
+
+The Gloo UI comes with an insights engine that automatically analyzes your Gloo Gateway setup for health issues. These issues are displayed in the UI along with recommendations to harden your setups. The insights give you a checklist to address issues that might otherwise be hard to detect across your environment.
+
+![Gloo UI insights]({{< versioned_link_path fromRoot="/img/ui-insights.png" >}})
+
+## Inventory
+
+The **Inventory** section provides an at-a-glance look at the health of registered clusters and discovered services that make up your Gloo Gateway environment.
+
+### Clusters
+
+On the **Clusters page**, review details of the cluster where Gloo Gateway is installed, such as insights, the health of the Gloo Gateway control and data planes, and applied routes, gateways, destinations, and policies.
+
+
+![Gloo UI Clusters]({{< versioned_link_path fromRoot="/img/ui-clusters.png" >}}) 
+
+## Traffic 
+
+The **Traffic** section provides an overview of deployed Gateways, routes, policies, and destinations. 
+
+
+### Gateways
+
+On the **Gateways** page, you can view the YAML configuration of gateway-related resources, such as gateway proxies, or MatchableHTTPGateways and MatchableTCPGateways. 
+
+To filter the list of resources, you can choose between the following options: 
+* Use the **Status** field to filter between healthy and unhealthy gateway resources.
+* Use the **Label key** and **value** fields to filter resources by their labels. 
+* Use the **Filter** to display the resource types that you are interested in. 
```

Comment:
maybe match the UI (but also fine if not, since it has enough detail to go by)
```suggestion
* Use the **Filter by** dropdown list to display the resource types that you are interested in. 
```